api/src/pcapi/settings.py

```diff
@@ -137,6 +138,10 @@
     os.environ.get("SUGGEST_REACTION_LONG_COOLDOWN_IN_SECONDS", 31 * 24 * 3600)
 )
 
+# USERS SESSION TIME
+PRO_SESSION_GRACE_TIME = timedelta(hours=int(os.environ.get("PRO_SESSION_GRACE_TIME", 1)))
+PRO_SESSION_LOGIN_TIMEOUT = timedelta(days=int(os.environ.get("PRO_SESSION_LOGIN_TIMEOUT", 14)))
```

Comment:
Détail: est ce que il ne serai pas pertinent de preciser l'unité dans les variables d'environnement ?
exemple:
```suggestion
PRO_SESSION_LOGIN_TIMEOUT = timedelta(days=int(os.environ.get("PRO_SESSION_LOGIN_TIMEOUT_IN_DAYS", 14)))
```